lumicks/pylake/tests/test_import_time.py

```diff
@@ -26,3 +27,33 @@ def test_disabling_capturing(report_line):
     report_line(
         f"Module import time: {np.mean(times):.2f} +- {np.std(times):.2f} seconds (N={repeats})"
     )
+
+
+def test_lazy_imports():
```

Comment:
I like that it's pinned by explicitly testing for it. Otherwise I would definitely see this being forgotten at some point.